common/main/mus.xml

```diff
@@ -89,13 +89,13 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 							<day type="sat">↑↑↑</day>
 						</dayWidth>
 						<dayWidth type="wide">
-							<day type="sun">Nettv Cako</day>
+							<day type="sun">Nettv'cako</day>
 							<day type="mon">Enhvteceskv</day>
 							<day type="tue">Enhvteceskv Enhvyvtke</day>
 							<day type="wed">Ennvrkvpv</day>
 							<day type="thu">Ennvrkvpv Enhvyvtke</day>
 							<day type="fri">Nak Okkoskv Nettv</day>
-							<day type="sat">Nettv Cakcuse</day>
+							<day type="sat">Nettv cak'cuse</day>
```

Comment:
This appears to be inconsistent titlecasing. Can you check?
Nak Okkoskv Nettv
vs
Nettv cak'cuse